screenshooter.go

```diff
@@ -47,8 +71,8 @@ func elementScreenshot(urlstr, sel string, res *[]byte) chromedp.Tasks {
 // Take a screenshot of the given post.
 func (ss *Screenshooter) Take(postURL url.URL, element string, filename string) error {
```

Comment:
I would change function signature:
```
Take(ctx context.Context, postURL url.URL, element string, w io.Writer) error 
```

let te caller decide which context and destination to use without opinions

Reply:
+1 on taking a `io.Writer` rather than a filename... it makes testing much simpler too!

Reply:
Oh yes, this should definitely be an `io.Writer`, thanks! :bowing_man: 

I'll keep the context out of the signature for now, since it is but an implementation detail of the `chromedp` library. If it starts to be more useful (for instance, to enable more flexible configuration), I'll add it to the signature. Sticking with the YAGNI mindset on this one. :stuck_out_tongue:

Reply:
Done! Looking good, if I can say so myself! :laughing: